t/01-compile-check-all.t

```diff
@@ -20,12 +20,24 @@ use FindBin;
 unshift @INC, "$FindBin::Bin/lib", "$FindBin::Bin/../lib";
 
 use Test::Strict;
+use Mojo::File;
 
 $Test::Strict::TEST_SYNTAX   = 1;
 $Test::Strict::TEST_STRICT   = 1;
 $Test::Strict::TEST_WARNINGS = 1;
-$Test::Strict::TEST_SKIP     = [
-    # skip test module which would require test API from os-autoinst to be present
-    't/data/openqa/share/tests/opensuse/tests/installation/installer_timezone.pm'
-];
```

Comment:
Perl didn't like this?

```
$Test::Strict::TEST_SKIP     = [
    # skip test module which would require test API from os-autoinst to be present
    't/data/openqa/share/tests/opensuse/tests/installation/installer_timezone.pm',
    # problem: when tidy test fails - it leaves .tdy files, so we should ignore them
    Mojo::File->new->list_tree->grep(qr/\.tdy$/)->map('to_rel')->map('to_string')->to_array,
    # docker tests are in bash
    glob "t/docker/*.t",
];
```

Otherwise I'd prefer it over introducing 2 new variables and an additional push.